boards/ek-lm4f120xl/Makefile.features

```diff
@@ -0,0 +1,5 @@
+FEATURES_PROVIDED += cpp
+#FEATURES_PROVIDED += periph_adc
```

Comment:
Then just delete this line for now and include it once you include it. (same for the implementation file...).